includes/upgradecheck.php

```diff
@@ -398,13 +398,6 @@ function pmpro_db_delta()
 		   `user_id` bigint(20) unsigned NOT NULL,
 		   `membership_id` int(11) unsigned NOT NULL,
 		   `code_id` bigint(20) unsigned NOT NULL,
-		   `initial_payment` decimal(18,8) NOT NULL,
```

Comment:
Let's leave this section here until we handle migration work, wouldn't want to run it and break your tables when switching between branches

Reply:
Just added this back